site/_posts/2017-09-29-miqldap-to-sssd.md

```diff
@@ -0,0 +1,315 @@
+---
+title: "MiqLdap To SSSD conversion tool"
+author: jvlcek
+date: 2017-09-29
+comments: true
+published: true
+tags: tutorials
+---
+
+How does one user the *miqldap_to_sssd* conversion tool?
+
+# MiqLdap To SSSD Conversion Tool
+
+**Contents:**
+---------------------------------------------------------------------
+
+- **A brief Why before The How:**
+- **The How**
+- **Let's kick the tires**
+  + **Example 1:** The simple Case:
+  + **Example 2:** A case with a small complication:
+  + **Example 3:** A case with a different small complication:
+- **The other miqldap_to_sssd arguments**
+- **Debugging Failed Conversions**
+
+**A brief Why before The How:**
+=====================================================================
+
+Why did the *miqldap_to_sssd* conversion tool come to be?
+
+MiQ is delivered with the built in LDAP client, *MiqLdap*. Maintaining our own LDAP
```

Comment:
First time mentioning MiQ, I think it's better to say "ManageIQ (MiQ) is delivered with ..."

Reply:
👍